src/components/players/details/games/index.module.scss

```diff
@@ -0,0 +1,53 @@
+@import 'src/scss/variables.scss';
+
+
+.game {
+    margin-bottom: 25px;
+}
+
+header {
+    display: flex;
+    align-items: center;
+    height: 40px;
+    font-family: arial;
+    font-size: 14px;
+    color: $grey5;
+
+    &:first-child {
+        padding-left: 7px;
+    }
+
+    div {
+        padding: 3px;
+    }
+
+    img {
+        display: inline-block;
+        padding: 3px;
+    }
+}
+
+.date {
+    display: flex;
+    margin-left: auto;
+    text-align: right;
+    font-size: 11px;
+    color: $grey4;
+}
+
+.titleContainer {
+    display: flex;
+    flex: 1;
+    justify-content: space-between;
+}
+
+.resultsContainer {
+    display: flex;
+    flex: 1;
+    // justify-content: space-between;
+    background-color: lightyellow;
+}
+
+section {
+    display: flex;
```

Comment:
Should also have `flex-direction: column;`

<img width="623" alt="Screen Shot 2020-04-05 at 2 16 46 PM" src="https://user-images.githubusercontent.com/623755/78506533-1b719280-7748-11ea-8bdf-2c241b80219d.png">